### PR TITLE
QF-1609 Add coronation emblem

### PR DESCRIPTION
**Pull request by SawNawDfE**

### CODE REVIEW

File: src/SFA.DAS.Courses.Api/ApiResponses/GetStandardResponse.cs [hunk 1]
```diff
@@ -47,6 +48,7 @@ public class GetStandardResponse
                 Status = source.Status,
                 SearchScore = source.SearchScore,
                 Title = source.Title,
+                CoronationEmblem = source.CoronationEmblem,
```
[the remark this answers is not in the document]
Comment (SawNawDfE) [reply src/SFA.DAS.Courses.Api/ApiResponses/GetStandardResponse.cs]: I've removed this, checked the other endpoints and it seems to work

**Review by ChrisJWoodcock — APPROVED** [review]

**Comment by SawNawDfE** [comment]
@ChrisJWoodcock The API project's unit tests won't run locally, and the failing test you mentioned is one of them. It started failing the last commit which removed the CoronationEmblem field from `GetStandardResponse`, and the test does check an object of that type for equivalency.

**Review by ChrisJWoodcock — APPROVED** [review]

File: src/SFA.DAS.Courses.Database/Tables/Standard.sql [hunk 2]
```diff
@@ -9,6 +9,7 @@ CREATE TABLE [dbo].[Standard]
     [VersionLatestEndDate] DATETIME NULL,
 	[Title] VARCHAR(1000) NOT NULL,
 	[Level] INT NOT NULL,
+    [CoronationEmblem] BIT NOT NULL DEFAULT 0,
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.Courses.Database/Tables/Standard.sql:12]: This column should be moved to the end of the table definition as during a DacPac deployment that will not require the rebuilding of the table and all the associated indexes - in this particular example it does not matter much - but it is recommended as a 'default' as on tables with lots of data that can cause issues with the deployments timing out

Comment (SawNawDfE) [reply src/SFA.DAS.Courses.Database/Tables/Standard.sql]: Done

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**